### PR TITLE
Add minimal support for config to compose

### PR DESCRIPTION
**Pull request by afbjorklund**

### CODE REVIEW

**Comment by fahedouch** [comment]
Thank you @afbjorklund, 
Can you add a test plz.

**Comment by AkihiroSuda** [comment]
test-integration-docker-compatibility failing

```
=== RUN   TestConfig
184
    compose_config_test.go:49: assertion failed: error is not nil: expected "services:\n  hello:\n    image: docker.io/nginx\n    networks:\n      default: null\nnetworks:\n  default:\n    name: testproject_default\n", got "services:\n  hello:\n    image: docker.io/nginx\nversion: '3.1'\n\n": services:
185
          hello:
186
            image: docker.io/nginx
187
        version: '3.1'
 ```

File: cmd/nerdctl/compose_config_test.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+package main
+
+import (
+	"testing"
+
+	"github.com/containerd/nerdctl/pkg/testutil"
+)
+
+func TestConfig(t *testing.T) {
```
Comment (AkihiroSuda) [inline cmd/nerdctl/compose_config_test.go:25]: TestComposeConfig

Comment (AkihiroSuda) [reply cmd/nerdctl/compose_config_test.go:25]: Also please squash commits

File: cmd/nerdctl/compose_config_test.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	base := testutil.NewBase(t)
+
+	var dockerComposeYAML = `
+version: '3.1'
+
+services:
+  hello:
+    image: docker.io/nginx
```
Comment (AkihiroSuda) [inline cmd/nerdctl/compose_config_test.go:33]: Invalid image name

File: cmd/nerdctl/compose_config_test.go [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  default:
+    name: testproject_default
+`
+
+	comp := testutil.NewComposeDir(t, dockerComposeYAML)
+	defer comp.CleanUp()
+
+	project := "testproject"
+	base.ComposeCmd("-f", comp.YAMLFullPath(), "-p", project, "config").AssertOutExactly(expectedOutput)
+}
```
Comment (AkihiroSuda) [inline cmd/nerdctl/compose_config_test.go:50]: `AssertOutContains(“hello:“)` might be enough

File: cmd/nerdctl/compose_config_test.go (hunk 2 above)
Comment (fahedouch) [reply cmd/nerdctl/compose_config_test.go:33]: @afbjorklund  list of image that you may use https://github.com/containerd/nerdctl/blob/1f817eb920470c491ba350c10f6df4eacaf7c0cd/pkg/testutil/testutil_linux.go#L26

**Comment by AkihiroSuda** [comment]
ping @afbjorklund

File: cmd/nerdctl/compose_config_test.go (hunk 3 above)
Comment (afbjorklund) [reply cmd/nerdctl/compose_config_test.go:38]: Done

File: cmd/nerdctl/compose_config_test.go (hunk 2 above)
Comment (afbjorklund) [reply cmd/nerdctl/compose_config_test.go]: Done

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**